src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -105,18 +116,18 @@
      * @return the number of completed assessments for the specified correction round of an exam exercise
      */
     @Query("""
-               SELECT COUNT(DISTINCT p)
-               FROM StudentParticipation p WHERE p.exercise.id = :exerciseId AND
-                            (SELECT COUNT(r)
-                            FROM Result r
-                            WHERE r.assessor IS NOT NULL
-                                AND r.rated = TRUE
-                                AND r.submission = (select max(id) from p.submissions)
-                                AND r.submission.submitted = TRUE
-                                AND r.completionDate IS NOT NULL
-                                AND (p.exercise.dueDate IS NULL OR r.submission.submissionDate <= p.exercise.dueDate)
-                                AND NOT EXISTS (select prs from p.results prs where prs.assessor.id = p.student.id)
-                            ) >= (:correctionRound + 1L)
+            SELECT COUNT(DISTINCT p)
+            FROM StudentParticipation p WHERE p.exercise.id = :exerciseId
+            AND (p.testRun = FALSE OR p.testRun IS NULL)
```

Comment:
```suggestion
            AND p.testRun = FALSE
```

Reply:
fixed in 139c8fb547deadc2b10c0e497d91389dcb5f736c